src/rpc/mining.cpp

```diff
@@ -262,7 +262,9 @@ static UniValue getmininginfo(const JSONRPCRequest& request)
 static UniValue prioritisetransaction(const JSONRPCRequest& request)
 {
             RPCHelpMan{"prioritisetransaction",
-                "Accepts the transaction into mined blocks at a higher (or lower) priority\n",
+                "Accepts the transaction into mined blocks at a higher (or lower) priority.\n"
+                "\nNote that prioritizing a transaction could leak privacy, through both\n"
```

Comment:
hm, there could be a privacy leak even if the txn isn't new. Even if it has propagated the network, if you `prioritisetransaction` it could cause the rebroadcast logic to pick up the txn & rebroadcast it sooner than it would have otherwise. this would indicate to a spy node that this txn is of special interest to you, thus leaking privacy.